lubridate.Rmd

```diff
@@ -296,81 +332,101 @@ int_season_autumn <- interval(season_19_autumn, season_19_winter - minutes(1))
 int_season_winter <- season_19_winter %--% (season_20_spring - minutes(1))
 ```
 
+Note that the `%--%` operator in the last line is similar to the calls of the `interval()` function in the lines above.
+
 ### Groupings
-What follows is preliminary step working with group_bys and renames in order to then apply the findings to seasons again. 
+What follows is an intermediate step working with `group_by()`s and `rename()`s. Subseqeuntly the findings towards seasons will be apllied again. 
```

Comment:
What follows is an intermediate step working with `group_by()`s and `rename()`s. Subseq**ue**ntly the findings towards seasons will be ap**p**lied again.

Reply:
Okay, that the inline comments are put here in front is a little akward.

I had a look into the diff oft he RMd file. Since there are too many changes to write about everyone of it.

I think regarding code, you have a lot more changes than I have done. E.g. the solution to suprress the libraries warnings = FALSE in {} is a solution, I have not come up with. I think I added more explaining to the code at the beginning and end oft he code chunks, but you have also added an explanation for e.g. stamp(). Also, I think you have found much more typos and wrong expressions than I have.

Apart from that we have taken a different approach with some things. E.g. I made the spring 2020 and summer 2020 dates as an unordered list. You have done it as a table. I think your solution is better here. 

Furthermore, I liked the Wikipedia link(s) you added to explain some of the topics.

Just as you mentioned, this took also quite some time for me,  7 hours or more. I am not quite sure, if this will be mergable.